README.MD

```diff
@@ -6,6 +6,7 @@
 - Yang Qian (Tech Lead)
 
 ## 2024 Team Members
+- Jack Haddad
```

Comment:
Hi Jack,

Can you please add your speciality or interest next to your name? For ex. Back-end, Front-end, Design, Architecture. You can add multiple interests if you'd like.

Reply:
Of course, I have added them in as requested :)